account_migrations/migrate.py

```diff
@@ -0,0 +1,327 @@
+import json, os
+from datadog import api 
+"""
+This script uses the datadog HTTP API to copy dashboard definitions from a source to a destination.
+TODO: switch from prints to logs
+TODO: compartmentalize by type?
+"""
+
+key_env_var_prefix_map = {
+        'https://app.datadoghq.com/':'APP',
+        'https://ddstaging.datadoghq.com/':'DDSTAGING',
+        'https://dd.datad0g.com/':'DD',
```

Comment:
Not sure these URLs are to be known by the public at ALL 😨  worth checking with older dogs.

Reply:
I'm an old dog. I guarantee they are now public.

Reply:
They were already.